.github/workflows/pull-requests.yml

```diff
@@ -590,23 +589,144 @@ jobs:
            fi
           done
 
-      - name: Request clusters info
+      - name: Check Global L0 nodes health
         run: |
-          sleep 5
-          echo "Dag L1"
-          curl http://127.0.0.1:9100/cluster/info
-          echo ""
-          
-          echo "Currency L0"
-          curl http://127.0.0.1:9400/cluster/info
-          echo ""
-          
-          echo "Currency L1"
-          curl http://127.0.0.1:9700/cluster/info
-          echo ""
+          for ((i = 1; i <= 12; i++)); do
+            if curl -s http://127.0.0.1:9000/node/info | grep -o "\"state\":\"Ready\"" >> /dev/null; then
+              echo "Global L0 health and Ready"
```

Comment:
*healthy

Reply:
I would just say that they are Ready actually. We are not checking healthiness as a different kind of check from status being Ready, so it always will be both or none.